models/people.js

```diff
@@ -0,0 +1,47 @@
+module.exports = function(sequelize, DataTypes) {
+    var People = sequelize.define("People", {
```

Comment:
Same as with Locations/Location in locations.js and also use const/let throughout